### ISSUE TITLE
Navigation bar improvements

### ISSUE THREAD

**Issue by mdelozzo** [openturns/www#69]
I used to access the [API documentation](https://openturns.github.io/openturns/latest/user_manual/user_manual.html) and I would appreciate to find a shortcut in the navigation bar which is currently:
<img width="406" alt="image" src="https://user-images.githubusercontent.com/53711375/182082627-8599deb6-7da0-453d-bb1c-12a39f2b7b02.png">

More generally, concerning the navigation bar, I propose some improvements based on my OpenTURNS user experience:
- remove _Get it_: once OpenTURNS installed, this tab is no longer useful; this page is only need once and could be referenced in [the home page](https://openturns.github.io/www/) with a button or something like that.
- rename _Modules_ to _Plugins_: it seems that this page contains only OpenTURNS plugins while _modules_ often refers to _Python modules_ of the documented library.
- make _Chat_, _Forum_ and _Bugs_ sub-tabs of a new tab named _Community_ (by the way, _Bugs_ is missing in the [navigation bar of the home page](https://openturns.github.io/www/).
- take advantage of the space freed up to add _API_, _Examples_ and _Theory_ so that these pages can be easily accessible from anywhere.

Remark: the URL https://openturns.github.io does not redirect to the [home page](https://openturns.github.io/www/) (contrarily to https://openturns.github.io/www/) but to the [documentation page](https://openturns.github.io/openturns/latest/contents.html).

**Comment by jschueller** [comment]
I dont think we mention swig submodules anywhere in the user doc, so "module" is not ambiguous.
"Plugins" usually suggests a different mecanism than independent libraries in top of openturns, so module is okish at least to designate the Python part.

**Comment by jschueller** [comment]
@josephmure @mbaudin47 I transfered the issue here btw
